canbench-bin/src/lib.rs

```diff
@@ -261,11 +263,14 @@ fn init_pocket_ic(
     canister_wasm_path: &PathBuf,
     stable_memory_path: Option<PathBuf>,
     init_args: Vec<u8>,
+    show_canister_output: bool,
 ) -> (PocketIc, Principal) {
     // PocketIC is used for running the benchmark.
     // Set the appropriate ENV variables
     std::env::set_var("POCKET_IC_BIN", path);
-    set_env_var_if_unset("POCKET_IC_MUTE_SERVER", "1");
+    if !show_canister_output {
+        set_env_var_if_unset("POCKET_IC_MUTE_SERVER", "1");
+    }
```

Comment:
@mraszyk Will this work as expected? What does PocketIC server do if no environment variable is set? Does it print logs or not? I seem to recall there's some subtle behaviour with this.

Reply:
Actually I was just reviewing https://github.com/dfinity/canbench/pull/86/files where Andriy added instructions how to run this and there is the output we expect. So I assume it works as we need.

Reply:
The environment variable `POCKET_IC_MUTE_SERVER` is read by the PocketIC (Rust) library, not the server. The behavior of the environment variable is as follows: no `POCKET_IC_MUTE_SERVER` or `POCKET_IC_MUTE_SERVER` set to an empty string means debug prints appear (just like any other stdout/stderr output) while a non-empty value of `POCKET_IC_MUTE_SERVER` means that stdout/stderr are redirected to /dev/null.